app/src/main/java/org/mozilla/reference/browser/ext/Context.kt

```diff
@@ -23,3 +31,26 @@ val Context.components: Components
 
 fun Context.getPreferenceKey(@StringRes resourceId: Int): String =
     resources.getString(resourceId)
+
+/**
+ *  Shares content via [ACTION_SEND] intent.
+ *
+ * @param text the data to be shared  [EXTRA_TEXT]
+ * @param subject of the intent [EXTRA_TEXT]
+ * @return true it is able to share false otherwise.
+ */
+fun Context.share(text: String, subject: String = ""): Boolean {
+    return try {
+        val intent = Intent(ACTION_SEND).apply {
+            type = "text/plain"
+            putExtra(EXTRA_SUBJECT, subject)
+            putExtra(EXTRA_TEXT, text)
+            flags = FLAG_ACTIVITY_NEW_TASK
+        }
+        startActivity(intent)
+        true
+    } catch (e: ActivityNotFoundException) {
+        Log.log(WARN, "Reference-Browser")
```

Comment:
I may have missed this before: The message "Reference-Browser" is not really helpful here. :)

How about something like:

```Kotlin
Logger.warn("No activity to share to found", e)
```

Reply:
Uff, I didn't realise that I haven't pass e as a parameter. Thanks